.github/workflows/linear.yml

```diff
@@ -0,0 +1,26 @@
+name: Check for "linear" in pull request body
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+
+jobs:
+  check_linear_in_pr_body:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for "linear" in the pull request body
+        run: |
+          pr_body=$(curl -s \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.number }} | jq -r .body)
+
+          echo "Pull Request Body: $pr_body"
+
+          if echo "$pr_body" | grep -iq "linear"; then
```

Comment:
You should not need to paste the entire link. Linear can find the GH PR just by pasting the ticket id in the body.

There must be a gh action that does this already properly

Reply:
yeah this is not the way to do it, also we want external contributions and this would fail those?

Reply:
I can use an existing github action instead

Re: external contributions - I thought about this and yes, it will fail, but this is an optional rule that serves more as a reminder. How much would we want to optimize for it? I could have a list of known github usernames of employees if we'd like. I just want to get something out first